docs/docs.css

```diff
@@ -0,0 +1,62 @@
+@import url(../stylesheets/index.css);
+.nav {
+  background-color: var(--primary-lighter);
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.left-nav {
+  font-size: 1.5rem;
+  font-weight: 900;
+  padding-left: 5rem;
+}
+.right-nav {
+  padding-right: 2rem;
+  gap: 2rem;
+  font-weight: 700;
+}
+
+main {
+  margin: 0 5rem;
+  padding-bottom: 5rem;
+  flex-direction: column;
+  align-items: center;
+}
+
+.logo {
+  justify-content: center;
+  margin: 1rem;
+}
+
+img {
```

Comment:
Instead of using this you can use a class named as docs-img and pass it these attributes or use the responsive image component which you built up recently if it fits the purpose

Reply:
made changes!